assets/app/view/spreadsheet.rb

```diff
@@ -96,14 +144,62 @@ def render_title
       ]
     end
 
+    def sort_order_icon
+      return '(⬇️)' if @spreadsheet_sort_order == 'ASC'
+
+      '(⬆️)'
+    end
+
+    def mark_sort_column(sort_by)
+      Lib::Storage['spreadsheet_sort_by'] = sort_by
+      store(:spreadsheet_sort_by, sort_by)
+      *@spreadsheet_sort_by = sort_by
```

Comment:
store will set the variable and rerender, you should not mutate the variable

Reply:
yep!